provider_run.go

```diff
@@ -437,6 +438,9 @@ func runSQL(ctx context.Context, db database.DBTxConn, m *Migration, direction b
 		statements = m.sql.Down
 	}
 	for _, stmt := range statements {
+		if p.cfg.verbose {
+			p.cfg.logger.Printf("Excuting statement: %s", stmt)
```

Comment:
I'm not sure I want these in my logs when enabling verbose logging. I.e., I care about the final output of the migration, and less about the individual statements. This almost feels more like debugging.

Reply:
Tricky then, because this is exactly the same as https://github.com/pressly/goose/blob/512a4968f549babc095a3ffe29c31325ba6d090c/migration_sql.go#L38

Here, I've just added the same behaviour for the provider as the goose usage without provider.

> This almost feels more like debugging.

Not really, since this is executed in an ideal world under some CD tooling, by any team, if would be nice to ensure that we have enough logs to assess a situation.
We had an issue where adding 40 columns, only 14 where added. We discovered why, but it would have helped to have the Goose `Executing Statement` log. 

For me, this is `verbose` logging behaviour that is supposed to be handled the right way by the provider. If, what is `WithVerbose` used for ?

Reply:
Okay fair, I think I can get behind this.